### PR TITLE
Days count wrong when birthday is not a Leap year

### LINKED ISSUE #3: Days count wrong when birthday is not a Leap year
Ex: birthday is 1991-10-27 
NIC : 913014***v

When I parse above nic, birthday is 1991-10-28;

I could fix this editing Parser.php as follow.

```
buildBirthDateObject(string $full_number, int $year) {
...
if (date('L', mktime(0, 0, 0, 1, 1, $year)) == 1) {
      --$birth_days_since;
    } else {
      $birth_days_since -= 2;
    }
}
...
```

Is there any other trick to fix this?

### PR DESCRIPTION
**Pull request by malinga91**

### CODE REVIEW

**Comment by malinga91** [comment]
> Hi @malinga91 - Please feel free to send a Pull-Request.
> 
> In `if (date('L', mktime(0, 0, 0, 1, 1, $year)) == 1) {`, I think it's better to use `if (date('L', mktime(0, 0, 0, 1, 1, $year)) === "1") {` as it provides type safety (even though by context, it should be safe with `==` comparator as well)

File: src/Parser.php [hunk 1]
```diff
@@ -86,7 +86,7 @@ private function buildBirthDateObject(string $full_number, int $year) {
       $this->data_components['gender'] = 'M';
     }
 
-    if (date('L', mktime(0, 0, 0, 1, 1, $year)) == 1) {
+    if (date('L', mktime(0, 0, 0, 1, 1, $year)) === "1")
```
Comment (Ayesh) [inline src/Parser.php]:
```suggestion
    if (date('L', mktime(0, 0, 0, 1, 1, $year)) === "1") {
```

There is a missing curly brace at the end, which prevents the tests from being run.

File: composer.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "slwdc/nic-parser",
+    "name": "malinga91/nic-parser",
```
Comment (Ayesh) [inline composer.json]: Please do not change the package namespace.

**Review by Ayesh — CHANGES_REQUESTED** [review]
Hi @malinga91 - thanks for submitting changes. 

I marked one line that has a syntax error, which prevents the tests from being run. 
After that, could you please add some tests to make sure the ID numbers are correctly decoded?

Thanks.

**Comment by Ayesh** [comment]
Thanks @malinga91 - I have merged and released `v1.1.0` of the library including your changes.

**Pull request merged**